### ISSUE TITLE
stuck on attempting to create session

### ISSUE THREAD

**Issue by dudefriendcologny** [justin025/onthespot#88]
issue started when i added apple music account

**Comment by justin025** [comment]
Does closing and reopening the app / waiting fix the issue?

**Comment by dudefriendcologny** [comment]
no, ive also tried deleting and installing the app again but doesnt work

**Comment by justin025** [comment]
on windows your settings file is located at '%appdata%/onthespot/otsconfig.json' and on macos at '~/.config/onthespot/otsconfig.json'. If you delete this file the app will return to its default state.

As for apple music, perhaps your login cookie was not exported correctly. I changed the login mechanism to use your account token instead of the cookie as there seemed to be some confusion. You can access the latest exe at the bottom of this page https://github.com/justin025/onthespot/actions/runs/12511804040 and login instructions can be found here https://github.com/justin025/onthespot/blob/main/docs/USAGE.md

Please let me know if this works
